torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2463,6 +2486,36 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                # TODO: remove redundant method_tests() entries
                SkipInfo('TestOpInfo', 'test_duplicate_method_tests')),
            sample_inputs_func=sample_inputs_addmm),
+    OpInfo('addmv',
+           dtypes=floating_types(),
+           dtypesIfCPU=all_types_and_complex_and(torch.float16, torch.bfloat16),
+           dtypesIfCUDA=floating_types_and(torch.float16, torch.complex64, torch.complex128,
+                                           *[torch.bfloat16] if CUDA11OrLater else []),
+           dtypesIfROCM=floating_types_and(torch.half),
+           skips=(
+               # https://github.com/pytorch/pytorch/issues/55539 to track all issues for addmv
+               # failed with a large difference in tensor outputs for below tests
+               SkipInfo('TestCommon', 'test_variant_consistency_eager', dtypes=(torch.float32, torch.complex64)),
+               # AssertionError: UserWarning not triggered : Resized a non-empty tensor but did not warn about it.
+               SkipInfo('TestCommon', 'test_out', dtypes=(torch.float32,)),
+               # RuntimeError: Jacobian mismatch for output 0 with respect to input 0,
+               # numerical:tensor([[ 1.0000e+00,  0.0000e+00,  0.0000e+00, -1.8758e+19,  0.0000e+00]])
+               # analytical:tensor([[1., 1., 1., 1., 1.]])
+               SkipInfo('TestGradients', 'test_inplace_grad', dtypes=(torch.float64,)),
+               # RuntimeError: Gradients failed to compare equal for grad output = 1j.
+               # Jacobian mismatch for output 0 with respect to input 0,
+               # numerical:tensor([[0.+1.0000e+00j, 0.+3.7808e+13j, 0.-9.1581e+12j, 0.+3.7517e+13j,
+               # 0.-1.5335e+13j]])
+               # analytical:tensor([[0.+1.j, 0.+1.j, 0.+1.j, 0.+1.j, 0.+1.j]])
+               SkipInfo('TestGradients', 'test_inplace_grad', dtypes=(torch.complex128,)),
```

Comment:
`addmv`'s inplace variant doesn't support autograd, so `supports_inplace_autograd=False` should be used instead of skipping inplace grad tests.

Reply:
Thanks! I'll change it :)